### PR TITLE
Bug -  added null checks to Config in SlidableLayout

### PR DESCRIPTION
**Pull request by ErlingMK**
## Added / Fixed

- Added default value to `Config` property in `SlidableLayout`, and null checks where `Config` is used. 

## Todo List

- [ ] I have tested on an Android device.
- [ ] I have tested on an iOS device.
- [ ] I have added unit tests

>*Please add pictures / Gifs if possible*
>*Todo List can be checked by putting a `X` inside the brackets*
>*Remember to update our `wiki` after this PR is merged*

### CODE REVIEW

**Comment by ErlingMK** [comment]
Considered adding a validation value check to property which throws an exception if `Config` is set to null. Might be preferable?

File: src/DIPS.Xamarin.UI/Controls/Slidable/SlidableLayout.cs [hunk 1]
```diff
@@ -221,8 +221,9 @@ protected virtual void OnScrolled(double index)
         public static readonly BindableProperty ConfigProperty = BindableProperty.Create(
             nameof(Config),
             typeof(SliderConfig),
-            typeof(SlidableLayout));
-
+            typeof(SlidableLayout),
+            defaultValue: new SliderConfig(int.MinValue, int.MaxValue));
```
Comment (Illedan) [inline src/DIPS.Xamarin.UI/Controls/Slidable/SlidableLayout.cs]: Keep default to 0, 1
Else we might end up with insane values if fallback is used.

File: src/DIPS.Xamarin.UI/Controls/Slidable/SlidableLayout.cs [hunk 2]
```diff
@@ -77,8 +77,8 @@ private void Rec_PanUpdated(object sender, PanUpdatedEventArgs e)
             }
 
             var currentPos = m_startSlideLocation - e.TotalX;
-            m_lastId = currentId;
-            var index = Math.Max(Config.MinValue - 0.45, Math.Min(Config.MaxValue + 0.45, CalculateIndex(currentPos)));
+            m_lastId = currentId;
```
Comment (Illedan) [inline src/DIPS.Xamarin.UI/Controls/Slidable/SlidableLayout.cs]: Add an if to the top of this method and return if Config is null. Should not continue with NULL config. As other bugs might occur.

File: src/DIPS.Xamarin.UI/Controls/Slidable/SlidableContentLayout.cs [hunk 3]
```diff
@@ -50,11 +50,11 @@ protected override void OnScrolled(double index)
                 for (var i = index - itemCount; i <= index + itemCount; i++)
                 {
                     var iIndex = (int)Math.Floor(i);
-                    if (iIndex < Config.MinValue || iIndex > Config.MaxValue) continue;
+                    if (iIndex < Config?.MinValue || iIndex > Config?.MaxValue) continue;
```
Comment (Illedan) [inline src/DIPS.Xamarin.UI/Controls/Slidable/SlidableContentLayout.cs]: Add if at the top and return if Config is NULL.

**Pull request merged**